### PR TITLE
Fixed bugs in names of classes and methods.

### PR DESCRIPTION
**Pull request by zomberg**
| Q | A |
| --- | --- |
| Branch? | 2.7 |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | ~ |
| License | MIT |
| Doc PR | ~ |

It's related to case sensitive.

I changed only calls of names of called methods but not definition of methods because BC.

### CODE REVIEW

**Comment by zomberg** [comment]
After fixing CS suggestion from fabbot.io I got new error.
Code suggested by fabbot.io:
```php
$instance->setBar(($this->get('foo')->foo().(($this->hasParameter('foo')) ? ($this->getParameter('foo')) : ('default'))));
```

Code generated by PhpDumper:
```php
$instance->setBar(($this->get("foo")->foo() . (($this->hasParameter("foo")) ? ($this->getParameter("foo")) : ("default"))));
```
I renamed 'hasparameter' -> 'hasParameter' and get CS notification from fabbot.io - remove space and change type of quotes.

So PhpDumper generates code which cannot get CS verification by fabbot.io. I see 3 ways:
1) Revert changes related with fabbot.io CS.
2) Fix CS for generating dump.
3) Change fabbot.io

I think first is the best. What do you think the best way solve this problem?

**Comment by stof** [comment]
you should ignore CS changes asked by fabbot when they are about fixture files (I thought fabbot was ignoring these files, but it seems it does not)

**Comment by zomberg** [comment]
Thanks, @stof. Reverted CS fixes for fixture files.

**Comment by javiereguiluz** [comment]
👍 

Status: reviewed

@zomberg thanks for these fixes.  I wasn't aware about how many of these errors were on the code!

File: src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandlerSchema.php [hunk 1]
```diff
@@ -38,8 +38,8 @@ private function addSessionTable($tableName)
     {
         $table = $this->createTable($tableName);
         $table->addColumn('sess_id', 'string');
-        $table->addColumn('sess_data', 'text')->setNotNull(true);
-        $table->addColumn('sess_time', 'integer')->setNotNull(true)->setUnsigned(true);
+        $table->addColumn('sess_data', 'text')->setNotnull(true);
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandlerSchema.php]: I'd rather change the method name here, WDYT?

Comment (nicolas-grekas) [inline src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandlerSchema.php]: hum ok, it's in doctrine, not our code... then I prefer our way :)

Comment (zomberg) [inline src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandlerSchema.php]: For me Symfony way is better too. But I think that we must use it as in declaration.

Comment (nicolas-grekas) [inline src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandlerSchema.php]: We *must* not, at least not technically (proof: it works without any issue)
Did you try submitting on doctrine?

Comment (zomberg) [inline src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandlerSchema.php]: Ok, I will revert this change and create pull-request to Doctrine. Will it ok?

Comment (nicolas-grekas) [inline src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandlerSchema.php]: yes for me

Comment (zomberg) [inline src/Symfony/Bridge/Doctrine/HttpFoundation/DbalSessionHandlerSchema.php]: Reverted this renaming.

**Comment by nicolas-grekas** [comment]
Thank you @zomberg.

**Pull request closed without merging**